src/components/List/List.jsx

```diff
@@ -6,9 +6,11 @@ import { useFirebase } from '../../hooks/useFirebase';
 const List = () => {
   const token = localStorage.getItem('token');
 
-  const { getAll } = useFirebase(token);
+  const { getAll } = useFirebase();
 
-  const [value, loading, error] = useCollection(getAll());
+  const [value, loading, error] = useCollection(
+    getAll().doc(token).collection('data'),
```

Comment:
what do you think if you create a `const data = getAll().doc(token).collection('data')` 

and then you put:

const [value, loading, error] = useCollection(data)

but you can put another name instance of data more descriptable

Reply:
Thanks, it has been solved.👍🏻